apps/backend/app/routers/templates.py

```
@@ -17,6 +45,71 @@ async def list_templates(
     return [doc async for doc in cursor]
 
 
+@router.post("", status_code=201)
+async def create_new_template(
+    body: CreateTemplateRequest,
+    current_user: dict = Depends(require_role("admin")),
```

Ruff (0.15.7)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)